src/app/services/baw-api/harvest/harvest.service.spec.ts

```diff
@@ -33,4 +39,34 @@ describe("HarvestsService", () => {
     harvestId, // harvest
     5 //project
   );
+
+  beforeEach((): void => {
+    spec = createService();
+    shallowSpectator = createShallowService();
+  });
+
+  function setup() {
+    const mockHarvestApi: BawApiService<Harvest> = spec.inject<BawApiService<Harvest>>(BawApiService);
+    spyOn(mockHarvestApi, "httpPatch").and.callFake((_path, _body, _options) => of());
+
+    return mockHarvestApi;
+  }
+
+  describe("updateName", () => {
+    it("should send a patch request to the server with the name property only as the body", () => {
+      const mockHarvestApi = setup();
+      const newHarvestName = "this is a test name";
+      const mockHarvest = new Harvest(generateHarvest());
+
+      shallowSpectator.service.updateName(mockHarvest, newHarvestName);
+
+      expect(mockHarvestApi.httpPatch).toHaveBeenCalledWith(
+        `/harvests/${mockHarvest.id}`,
+        // disable eslint for this line because the baw-server is expecting this object's
```

Comment:
baw-server actually expects snake_case - the comment is incorrect

Naming is so because it hasn't passed through our case converter in the http interceptor api yet